app/test/process_proposal_test.go

```diff
@@ -154,59 +270,6 @@ func TestProcessProposal(t *testing.T) {
 	}
 }
 
-func TestProcessProposalWithParityShareNamespace(t *testing.T) {
```

Comment:
moved this test up to the other test, see comment there to see why we probably don't even need this test unless we have some code that let's us bypass these checks in prepare proposal #233